AggregateMetrics/AggregateMetrics/AzureWebApp/Implementation/CounterFactory.cs

```diff
@@ -9,42 +9,57 @@
     internal class CounterFactory
     {
         /// <summary>
+        /// Gets metric alias to be the value given by the user.
+        /// </summary>
+        /// <param name="counterName">Name of the counter to retrieve.</param>
+        /// <param name="reportAs">Alias to report the counter.</param>
+        /// <returns>Alias that will be used for the counter.</returns>
+        private string GetCounterReportAsName(string counterName, string reportAs)
```

Comment:
Since you are sanitizing name in module - you need to move this logic to the module as well